docs/kb/advanced-certificates.md

```diff
@@ -63,3 +63,98 @@ LS0tLS1CRUd......==
+
+In order to import your trusted root certificate, please follow the steps outlined below.
+
+### Assumptions
+
+* Pre-download you root certificates from your internal certificate, which must be .PEM encoded
+
+In the example, the downloaded root certificate file is named `ca.crt`
+
+### Steps
+
+Step 1 - Transfer or copy the contents of the root certificate to the VMware Event Broker Appliance and create a new kubernetes configMap from that file.
+
+```console
+kubectl -n vmware-system create cm ca-cert --from-file ca.crt
+```
+
+Step 2 - Undeploy the VMware Event Router and delete the current configuration secret.
+
+```console
+kubectl -n vmware-system delete -f /root/config/event-router-k8s.yaml
+kubectl -n vmware-system delete secret event-router-config
+```
+
+Step 3 - Edit `/root/config/event-router-k8s.yaml` and append the additional volumenMount and volume entry as shown in the snippet below.
+
+```console
+snip
+...
+          volumeMounts:
+            - name: config
+              mountPath: /etc/vmware-event-router/
+              readOnly: true
+            - name: ca-cert
+              mountPath: /etc/vmware-event-router/ssl
+              readOnly: true
+      volumes:
+        - name: config
+          secret:
+            secretName: event-router-config
+        - name: ca-cert
+          configMap:
+            name: ca-cert
+```
+
+Step 4 - Edit `/root/config/event-router-config.yaml` and append the additional certificates section to the end of the file as shown in the snippet below.
+
+```console
+snip
+...
+metricsProvider:
+  default:
+    bindAddress: 0.0.0.0:8082
+  name: veba-metrics
+  type: default
+certificates:
+  rootCAs:
+    - /etc/ssl/certs/ca-certificates.crt
+    - /etc/vmware-event-router/ssl/vc-ca.crt
+```
+
+> **Note:** If `insecureSSL` is set to `true`, please update that to `false` which will now ensure TLS verification is performed when connecting to your vCenter Server.
+
+Step 5 - Recreate the VMware Event Router configuration and redeploy the VMware Event Router deployment.
+
+```console
+kubectl -n vmware-system create secret event-router-config
+kubectl -n vmware-system apply -f /root/config/event-router-k8s.yaml
+```
+
+Step 6 - Verify the VMware Event Router pod is running successfully.
+
+```console
+kubectl -n vmware-system get pods
+NAME                                   READY   STATUS    RESTARTS   AGE
+tinywww-dd88dc7db-q6pzw                1/1     Running   0          166m
+veba-rabbit-server-0                   1/1     Running   0          167m
+veba-ui-8d6584f84-ckvwb                1/1     Running   0          166m
+vmware-event-router-7759d8bffc-45c92   1/1     Running   0          37m
+```
+
+Lastly, you can also look at the VMware Event Router log and ensure you are no should seeing the message `potentially insecure connection to vCenter`, which is what you would see if the connection was not trusted if you had TLS verified disabled.
```

Comment:
This is a bit confusing as the output you show is actually saying it's insecure. You might want to move that paragraph above to the intro section to show that this example/doc fixes this behavior. At the end of this section, you can show the output of the router again without this line being printed.

Reply:
Make sense.